firestore/add_data.rb

```diff
@@ -168,13 +168,38 @@ def update_server_timestamp project_id:
   # project_id = "Your Google Cloud Project ID"
 
   firestore = Google::Cloud::Firestore.new project_id: project_id
+  firestore.doc("cities/new-city-id").set(
```

Comment:
Is this an expected change?

Reply:
After speaking with @BenWhitehead offline, it looks like this is necessary but not sure how this was first merged without passing CI. Question for another day so long as tests pass.